assets/static/js/project.js

```diff
@@ -33,37 +33,3 @@ function switchDashboards(event) {
     // Hide the menu after selection
     $("#accordionSidebar.toggled #dashboards-menu").removeClass("show");
 }
-
-
-(function($) {
```

Comment:
Why did you delete this 😕??

Reply:
It's the function in which google analytics script was embedded. Its actions seems not applicable in our case.  With/without this function, we can show/hide dashboards, minimize and maximize the sidebar. I don't see where it applies unless you found something.